src/main/kotlin/org/jetbrains/kotlin/jupyter/repl.kt

```diff
@@ -38,7 +38,8 @@ class ReplCompilerException(val errorResult: ReplCompileResult.Error) : ReplExce
 }
 
 class ReplForJupyter(val scriptClasspath: List<File> = emptyList(),
-                     val config: ResolverConfig? = null) {
+                     val config: ResolverConfig? = null,
+                     val properties: RuntimeKernelProperties = RuntimeKernelProperties()) {
```

Comment:
Why does ReplForJupyter need RuntimeKernelProperties? They are not used within it